docs/provenance/provenance-en.ttl

```diff
@@ -0,0 +1,19 @@
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix dc: <http://purl.org/dc/terms/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix : <> .
+<https://w3id.org/ontouml/1.1.0> a prov:Entity;
+	 dc:title "OntoUML Metamodel Vocabulary";
```

Comment:
The title here also needs to be changed.

Reply:
Done.